Core/HLE/sceNetAdhoc.cpp

```diff
@@ -1637,7 +1640,7 @@ static int sceNetAdhocPtpOpen(const char *srcmac, int sport, const char *dstmac,
 			}
 			
 			// Valid Ports
-			if (!isPTPPortInUse(sport) && dport != 0) {
+			if (!isPTPPortInUse(sport) /*&& dport != 0*/) {
```

Comment:
need to add 1 more line under this to fix connection issue with Phantasy Star Portable 2 when using non-zero PortOffset:

if (sport == 0) sport = -(int)portOffset; //making sure sport is 0 when shifted